scripts/test/corelli/calibration/test_utils.py

```diff
@@ -104,24 +130,73 @@ def test_load_banks(self):
         self.assertAlmostEqual(workspace.readY(42)[0], 13297.0)
         DeleteWorkspaces(['jambalaya'])
 
-    def test_calculate_tube_calibration(self) -> None:
+    def test_peak_y_table(self) -> None:
+        # Mock PeakTable with two tubes and three peaks. Simple, integer values
+        def peak_pixels_table(table_name, peak_count, tube_names=None, pixel_positions=None):
```

Comment:
add return type `->TableWorkspace` to function signature

Reply:
I have included the annotation, although as a rule, I avoid type annotations in tests :sleepy: